### ISSUE TITLE
ImageJ ROI Converter again

### ISSUE THREAD

**Issue by DocObvious** [MouseLand/cellpose#288]
Hi  I am having a similar issue as #101 when I try to get ImageJ ROI text I get an error:

AttributeError: module 'cellpose.plot' has no attribute 'outlines_list'

I have tried to update cellpose via pip install cellpose --upgrade, but hasn't worked

Does anyone know what the correct code is to save the ROI for imageJ?

Thanks

**Comment by DocObvious** [comment]
With a little bit of digging around I found the correct code for anyone else having this same issue: the outline_list attribute is found in the utils module not the plot module. 
This is the code that works for me:

for idx in range(nimg):
    base = filenames[idx]
    outlines = utils.outlines_list(masks[idx])
    io.outlines_to_text(base, outlines)

You need to make sure that filenames is a list of the exact filesnames include file type for this to work.

**Comment by carsen-stringer** [comment]
I'm sorry can you say where you are getting the code from to create the outlines list? Is this from the GUI?

**Comment by DocObvious** [comment]
Hello
It was the outlines_list function that was causing me issues. I kept getting attribute errors for this when using the plot class. 
Therefore I did have a look at the GUI code to see where it got the outlines_list from when using that. Then also scoured the other classes base code to see if I could find an outlines_list function and found it in utils so I tried it and it worked perfectly.

Also just wanted to thank and congratulate you and your team on building an amazing tool it really is extremely accurate and you have done a great job of documenting everything too.
